core/src/main/kotlin/okuri/core/service/block/EntityBlockReferenceService.kt

```
@@ -0,0 +1,29 @@
+import org.springframework.stereotype.Service
+import java.util.*
+
+/**
+ *  This service layer is responsible for handling the management of block references from within an entity.
+ */
+@Service
+class EntityBlockReferenceService(
+    private val entityBlockReferenceRepository: EntityBlockReferenceRepository
+) {
+    fun attachBlock(request: AttachBlockRequest) {}
+    fun detachBlock(request: DetachBlockRequest) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)